manifests/2.12.0/opensearch-dashboards-2.12.0-test.yml

```diff
@@ -83,8 +83,6 @@ components:
   - name: assistantDashboards
     integ-test:
       test-configs:
-        - with-security
```

Comment:
Even after reading the description I dont get why are we removing the security test. Could you elaborate?

Thanks.

Reply:
assistantDashboards requires a config in system index, but we can not `put` doc into that index in our cypress test workflow as it requires a security .pem file to do so.

Reply:
If we want to do test against a Chatbot root agent, we have to set the config inside the system index. However when security enabled, we can not do that inside Cypress test as system_index setting is a static setting that can only be modified in the opensearch.yml file.

Reply:
So you mean in order to support tests with security you need additional config to be added to opensearch.yml?

Reply:
Yes, that should work too.

Reply:
Looks like a feature request. AFAIK `additional-cluster-configs` only supports the product under test (OpenSearch or Dashboards. Not both) 
https://github.com/search?q=repo%3Aopensearch-project%2Fopensearch-build+additional-cluster-configs&type=code&p=1

@peterzhuamazon Looks like time to open this issue https://github.com/opensearch-project/opensearch-build/issues/3589 :)

Reply:
Have sync up with @SuZhou-Joe and it seems we either implement #3589, which we dont have much time for 2.12, or remove the feature after @SuZhou-Joe discuss with @ylwu-amzn.

There is another option which is to comment out the `- with-security` and list it a `TODO` to be fixed in the next release.